code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -514,16 +538,49 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/L, metabolis
 /datum/reagent/peridaxon/overdose_crit_process(mob/living/L, metabolism)
 	L.apply_damages(2*REM, 6*REM, 6*REM)
 
+
+/datum/reagent/medicine/peridaxon_plus
+	name = "Peridaxon"
+	description = "Used to repair internal organs. Significant toxic and stun side effects."
+	color = "#C845DC"
+	overdose_threshold = REAGENTS_OVERDOSE/30
+	overdose_crit_threshold = REAGENTS_OVERDOSE_CRITICAL/25
+	custom_metabolism = REAGENTS_METABOLISM * 0.25
+	scannable = TRUE
+
+/datum/reagent/medicine/peridaxon_plus/on_mob_life(mob/living/L, metabolism)
+	L.apply_damage(10*REM, TOX) //50 toxin/unit
+	L.adjustHalLoss(300) //You do not get up while this is in you. At all.
```

Comment:
Give it a stun effect on mob add instead. That said, there's combat-med balance concerns on this.

Reply:
It's both NOT intended to be a combat med, and I didn't want to give it a stun, because stuns are counterable.

This one's also not obtainable by players in this PR yet, I agree with you about the balance concerns.

Reply:
Stun changed to 4 seconds/tick.

Combat med balance concerns left unaddressed because it's not acquireable as of now, and because when it becomes acquireable, it'll be medic-only on a points basis.